docs/team/ytingtan.md

```diff
@@ -35,7 +30,7 @@ Given below are my contributions to the project.
       * Rating is an integer between 1 and 10 inclusive.
       * It is a non-compulsory attribute of a candidate, as the recruiter might not have enough information to rate the candidate at the point of application. 
       * The Rating attribute can be used in three ways: 
-        1. Adding a new candidate `add n/John Doe e/john@mail.com p/10384280 s/Application Received rate/9` adds a new candidate with name `John Doe`, email `john@mail.com`, phone number `10384280`, status of `Application Received` with rating of `9`.
+        1. Adding a new candidate `add n/John Doe p/10384280 e/john@mail.com s/Application Received rate/9` adds a new candidate with name `John Doe`, phone number `10384280`, email `john@mail.com`, status of `Application Received` with rating of `9`.
```

Comment:
Here as well, you must update the example with all the compulsory fields for the add command.